app/src/main/java/com/example/frogweather/data/FrogWeatherApiService.kt

```diff
@@ -0,0 +1,26 @@
+package com.example.frogweather.data
+
+import com.squareup.moshi.Moshi
+import com.squareup.moshi.kotlin.reflect.KotlinJsonAdapterFactory
+import retrofit2.Retrofit
+import retrofit2.converter.moshi.MoshiConverterFactory
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+private val moshi = Moshi.Builder()
+    .add(KotlinJsonAdapterFactory())
+    .build()
+
+private val retrofit = Retrofit.Builder()
+    .addConverterFactory(MoshiConverterFactory.create(moshi))
+    .baseUrl(WEATHER_API_BASE_URL)
+    .build()
+
+interface FrogWeatherApiService {
+    @GET(WEATHER_API_PATH_URL)
```

Comment:
You can leave here hardcoded value, it is easy to see when we have a lot of calls and files